parts/k8s/kubernetesmastervars.t

```diff
@@ -119,7 +119,11 @@
     "cloudProviderRatelimitQPS": "[parameters('cloudProviderRatelimitQPS')]",
     "cloudProviderRatelimitBucket": "[parameters('cloudProviderRatelimitBucket')]",
     "useManagedIdentityExtension": "{{ UseManagedIdentity }}",
+{{if .HasVirtualMachineScaleSets}}
+    "useInstanceMetadata": "false",
```

Comment:
1.10 supports setting useInstanceMetadata to both true and false, so we should keep same logic: `"useInstanceMetadata": "{{ UseInstanceMetadata }}"`

Reply:
due to https://github.com/kubernetes/kubernetes/pull/62611, if `instanceMetadata` is enabled, master kubelet won't start. If it's disabled, it works fine. One possible workaround is to duplicate cloud config on the masters, so master has kubelet `vmType` as standard and controller-manager has vmss.

Reply:
Since this is experimental (we'll add docs later) let's change the above to `"useInstanceMetadata": "{{ UseInstanceMetadata }}"`. Experimenters pre-1.10.2 should specify `"useInstanceMetadata": "false"` in their api models until then.

This way we don't have to submit a follow-up PR after 1.10.2.